contracts/pool/LiquidityLocker.sol

```diff
@@ -30,9 +34,29 @@ contract LiquidityLocker is ILiquidityLocker {
         return IERC20(liquidityAsset).balanceOf(address(this));
     }
 
+    function setSecondaryLiquidityAsset(address _liquidityAsset) external {
+        secondaryLiquidityAssets[_liquidityAsset] = true;
+    }
+
+    function deleteSecondaryLiquidityAsset(address _liquidityAsset) external {
+        delete secondaryLiquidityAssets[_liquidityAsset];
+    }
+
+    function setSecondaryLiquidityAssets(
+        address[] calldata _liquidityAssets
+    ) external {
+        for (uint i = 0; i <= _liquidityAssets.length; i++) {
+            secondaryLiquidityAssets[_liquidityAssets[i]] = true;
+        }
+    }
+
+    function assetsExists(address _assetAddr) external returns (bool) {
```

Comment:
I guess function name could be **secondaryAssetExists**

`function secondaryAssetExists(address _assetAddr) external returns (bool) {`